source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -37,16 +37,26 @@ example with this shell script:
   SRV _mongodb._tcp.test2.test.build.10gen.cc localhost 27018
   SRV _mongodb._tcp.test2.test.build.10gen.cc localhost 27019
   SRV _mongodb._tcp.test3.test.build.10gen.cc localhost 27017
+  SRV _mongodb._tcp.test5.test.build.10gen.cc localhost 27017
+  SRV _mongodb._tcp.test6.test.build.10gen.cc localhost 27017
+  TXT test5.test.build.10gen.cc "connectTimeoutMS=300000&socketTimeoutMS=300000"
+  TXT test6.test.build.10gen.cc "connectTimeoutMS=200000"
+  TXT test6.test.build.10gen.cc "socketTimeoutMS=200000"
   EOF
 
   export LD_PRELOAD=`pwd`/resolv_wrapper_build/src/libresolv_wrapper.so
   export RESOLV_WRAPPER_HOSTS=hosts.txt
 
 You can debug the library by exporting ``RESOLV_WRAPPER_DEBUGLEVEL=3``.
 
-If you choose instead to configure a real name server with SRV records, adapt
-the records shown above to your domain name, and update the "uri" field in the
-YAML or JSON files in this directory with the actual domain.
+Please note that CWRAP does not return more than one record per type/host
+combination. It also does not understand TXT records, so your mileage may
+vary.
```

Comment:
It's time to give up and remove the CWRAP instructions entirely. CWRAP does not do what we need.

Reply:
Yeah, I agree. It's either that, or fixing CWRAP. I will remove the instructions from the README.